### PR TITLE
Minor simplifications

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

File: src/Validation/ConstraintViolation.php [hunk 1]
```diff
@@ -45,10 +45,7 @@ public function setSource( string $source ) {
 		$this->source = $source;
 	}
 
-	/**
-	 * @return mixed|null
-	 */
-	public function getConstraint() {
+	public function getConstraint(): string {
```
Comment (KaiNissen) [inline src/Validation/ConstraintViolation.php]: This method actually returns a validator object.

Comment (JeroenDeDauw) [inline src/Validation/ConstraintViolation.php]: Ugh... so this means we do not have a single test that executes this code path?!

**Comment by JeroenDeDauw** [comment]
Panic!!!!!!!!!!!!!!

**Comment by KaiNissen** [comment]
This method is also never used.

**Comment by JeroenDeDauw** [comment]
I'm killing it now

**Comment by JeroenDeDauw** [comment]
See #132

**Pull request merged**